.gitignore

```diff
@@ -10,3 +10,4 @@
 
 # rspec failure tracking
 .rspec_status
+.DS_Store
```

Comment:
and to add to this, the project's gitignore should be responsible for project's specific files not overall system's file imho